migrations/20191007-add_baseBranch_column_to_events.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}events`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.createTable(table, {
```

Comment:
`createTable` -> `addColumn` ?
Refs:
- https://sequelize.org/master/manual/migrations.html
- https://sequelize.org/master/class/lib/query-interface.js~QueryInterface.html#instance-method-addColumn

Reply:
That's true. Fixed to use `addColumn`.